be1-go/hub/mod.go

```diff
@@ -0,0 +1,6 @@
+// Package hub defines a Hub that is used for processing incoming messages
+// and replying back to the client.
+//
+// A concrete instance of a Hub may be an Organizer or a Witness. A baseHub
+// type contains the implementation common across both types.
+package hub
```

Comment:
The package also defines the sockets , would it make sense to cover all of that here?